### PR TITLE
send installID as a head with each API request

### PR DESCRIPTION
**Pull request by maxtaco**
- add support for looking at the user's updater.json config file
- test is coming as a separate job

### CODE REVIEW

File: go/libkb/constants.go [hunk 1]
```diff
@@ -51,11 +51,12 @@ var GregorServerLookup = map[RunMode]string{
...
 
 const (
-	ConfigFile  = "config.json"
-	SessionFile = "session.json"
-	DBFile      = "keybase.leveldb"
-	SocketFile  = "keybased.sock"
-	PIDFile     = "keybased.pid"
+	ConfigFile        = "config.json"
+	SessionFile       = "session.json"
+	UpdaterConfigFile = "updater.json"
```
Comment (patrickxb) [inline go/libkb/constants.go]: UpdaterFile to match `updater-file` and `updater_file`?

**Comment by patrickxb** [comment]
LGTM

File: go/libkb/constants.go (hunk 1 above)
Comment (maxtaco) [inline go/libkb/constants.go]: i'll change the two names in the match `UpdaterConfigFile`

**Pull request merged**